package/boot/uboot-rockchip/patches/102-rockchip-rk3328-Add-support-for-OrangePi-R1-Plus.patch

```diff
@@ -0,0 +1,519 @@
+--- a/arch/arm/dts/Makefile
++++ b/arch/arm/dts/Makefile
+@@ -109,6 +109,7 @@ dtb-$(CONFIG_ROCKCHIP_RK3308) += \
+ dtb-$(CONFIG_ROCKCHIP_RK3328) += \
+ 	rk3328-evb.dtb \
+ 	rk3328-nanopi-r2s.dtb \
++	rk3328-orangepi-r1-plus.dtb \
+ 	rk3328-roc-cc.dtb \
+ 	rk3328-rock64.dtb \
+ 	rk3328-rock-pi-e.dtb
+--- /dev/null
++++ b/arch/arm/dts/rk3328-orangepi-r1-plus-u-boot.dtsi
```

Comment:
Where is this file used?

Reply:
From https://wiki.st.com/stm32mpu/wiki/STM32MP15_U-Boot
> \<Device tree\>-u-boot.dtsi : addition for u-boot automatically included in build process

We tell u-boot to use ddr4 memory for this board here as the RK3328 supports different types of RAM (LPDDR3/DDR3/DDR4).